chapters/generative-models.md

```diff
@@ -112,7 +112,9 @@ What if we want to invoke this sampling process multiple times? We would like to
 We can use `function` to construct such complex stochastic functions from the primitive ones.
 
 ~~~~
-var sumFlips = function() { return flip() + flip() + flip() }
+var sumFlips = function () {
```

Comment:
I'd strongly prefer not to put a space between `function` and the parens (and between a function name and the arg parens). could you revert that in the PR?